frontend/packages/console-plugin-sdk/src/typings/dashboards.ts

```diff
@@ -131,6 +131,9 @@ namespace ExtensionProperties {
     /** The tab's ID which will be used as part of href within dashboards page */
     id: string;
 
+    /** NavSection to which the tab belongs to */
+    navSection: string;
```

Comment:
lets be more strict and specify values which are valid.
```suggestion
    navSection: 'home' | 'storage';
```